### PR TITLE
Split test_parallel_executor_seresnext to three unit test

### PR DESCRIPTION
**Pull request by chengduoZH**
The unit test random failed in CI system because of the timeout, but I can not reproduce on my developing machine, I guess that maybe that because of the CI system is not stable, because the CE test and Benchmark test don't have any abnormal.

So split test_parallel_executor_seresnext to three unit test and temporally disable reduce_and_allreduce test because of the random failure.

### CODE REVIEW

File: python/paddle/fluid/tests/unittests/CMakeLists.txt [hunk 1]
```diff
@@ -267,7 +267,7 @@ py_test_modules(test_parallel_executor_crf MODULES test_parallel_executor_crf)
 py_test_modules(test_parallel_executor_crf_auto_growth MODULES test_parallel_executor_crf_auto_growth ENVS FLAGS_allocator_strategy=auto_growth)
 py_test_modules(test_parallel_executor_fetch_feed MODULES test_parallel_executor_fetch_feed)
 set_tests_properties(test_parallel_executor_fetch_feed PROPERTIES TIMEOUT 450)
-set_tests_properties(test_parallel_executor_seresnext PROPERTIES TIMEOUT 740)
+set_tests_properties(test_parallel_executor_seresnext PROPERTIES TIMEOUT 940)
```
Comment (luotao1) [inline python/paddle/fluid/tests/unittests/CMakeLists.txt]: 这个应该不是根本原因吧。正常运行的时候没有跑那么长时间的。

Comment (chengduoZH) [reply python/paddle/fluid/tests/unittests/CMakeLists.txt]: 不确定是不是机器不稳定导致，因为我这边没办法复现这个问题，并且现在CE和Benchmark中都有se-resnext模型，也没有遇到有异常的

Comment (luotao1) [reply python/paddle/fluid/tests/unittests/CMakeLists.txt]: 1. 建议线下增大压测次数来复现。该单测随机挂的频率还是很高的。
2. 建议拆分单测来加速CI。目前该单测运行7分钟，整个CI单测25分钟，占3分之一时间。
http://ci.paddlepaddle.org/viewLog.html?buildId=141522&tab=buildLog&buildTypeId=Paddle_PrCiCoverage&logTab=tree&filter=all&_focus=16815#_state=84
```
[03:07:35]	100% tests passed, 0 tests failed out of 164
[03:07:35]	Total Test time (real) = 887.89 sec
[03:14:00]	RUN_TYPE=DIST    = 865.78 sec (11 tests)
[03:26:02]	RUN_TYPE=EXCLUSIVE    = 836.07 sec (18 tests)

[03:20:18] :	 [Step 1/1] 17/18 Test #418: test_parallel_executor_seresnext ..........   Passed  697.84 sec
```

**Review by luotao1 — APPROVED** [review]
LGTM

**Pull request merged**